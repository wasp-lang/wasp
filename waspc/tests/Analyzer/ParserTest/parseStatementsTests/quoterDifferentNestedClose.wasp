test DifferentNestedClose {=foo bar=} foo=}
