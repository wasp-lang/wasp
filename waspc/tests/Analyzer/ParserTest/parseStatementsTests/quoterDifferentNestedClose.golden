(AST
  (Decl@1:1-43 type=test name=DifferentNestedClose
    (Quoter@1:27-43 tag=foo
      {= bar=} =}
    )
  )
)
